src/main/content/index.html

```diff
@@ -106,6 +106,28 @@ <h1 class="white-text expressive-heading-06">{{t.index.welcome-kabanero}}</h1>
                 </li>
             </ul>
 
+            <div id="dev-tools-codready-install-steps">
+                <p>{{t.index.devtools.steps.codeready}}</p>
+                <ul>
+                    <li>
+                        {{t.index.devtools.steps.follow-steps}}
+                        <a target="_blank"
+                            href="/docs/ref/general/installation/installing-codeready-and-codewind">{{t.index.devtools.steps.codereadyInstall}}</a>
+                    </li>
+                </ul>
+            </div>
+
+            <div id="dev-tools-codready-installed" class="hide">
+                <p>{{t.index.devtools.steps.codeready}}</p>
+                <ul>
+                    <li>
+                        <a id="dev-tools-modal-codeready-link" target="_blank"
+                            href="">{{t.index.devtools.steps.go-to-codeready}}</a>
+                    </li>
+                </ul>
+            </div>
```

Comment:
this seems like too much duplication since the ` <p>{{t.index.devtools.steps.codeready}}</p>` line is the same in both. I would put do just another <li> that's hidden instead of a whole div like this.